packages/ses/test/shim-arraybuffer-transfer.test.js

```diff
@@ -0,0 +1,61 @@
+import test from 'ava';
+import '../index.js';
+
+lockdown();
+
+// The purpose of this test is to see if Array.prototype.transfer works
+// correctly enough on platforms like Node 18 or Node 20 that don't yet have
+// it natively, and so are testing the shim on those. On platforms where
+// Array.prototype.transfer is present, like Node 22,
+// we also run the same tests.Thus,
+// this test only tests the intersection behavior of the standard and
+// the shim. The shim does not yet support a `newLength` argument
+// larger than the original.
+//
+// TODO once the shim supports transfering to a larger length, we must
+// test that as well.
```

Comment:
I think this needs to be updated now :)

Reply:
Was already done, but I had not noticed the stale comment. Thanks. Fixed.